test/integration/cdi/example-driver/app/kubeletplugin.go

```diff
@@ -0,0 +1,207 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package app
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	cdiapi "github.com/container-orchestrated-devices/container-device-interface/pkg/cdi"
+	specs "github.com/container-orchestrated-devices/container-device-interface/specs-go"
+	"k8s.io/klog/v2"
+	drapbv1 "k8s.io/kubernetes/pkg/kubelet/apis/dra/v1alpha1"
+	plugin "k8s.io/kubernetes/test/integration/cdi/example-driver/kubeletplugin"
+)
+
+type pluginConfig struct {
+	cdiDir                 string
+	driverName             string
+	draAddress             string
+	pluginRegistrationPath string
+}
+
+type examplePlugin struct {
+	config             pluginConfig
+	resourceHandleKeys []string
+}
+
+var resourceHandleKeys = []string{"some", "another"}
+
+// newExamplePlugin returns an initialized examplePlugin instance.
+func newExampleDriver(config pluginConfig) (*examplePlugin, error) {
+	examplePlugin := &examplePlugin{
+		config: config,
+	}
+	examplePlugin.resourceHandleKeys = makePossibleResourceHandleKeys(resourceHandleKeys)
+	return examplePlugin, nil
+}
```

Comment:
misleading naming: newExample**Driver** returns example**Plugin**. Is it plugin or driver? Decide and name accordingly.

Reply:
To be honest, I don't know what is driver what is plugin....